object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -54,12 +69,14 @@ The next example configures Salesforce connector authentication through OAuth, t
 ----
 
 [[private_os]]
-== Define a Private Object Store
+== Defining a Private Object Store
 
-* For cases where shared state is a security risk, you should use a private Object Store.
-* For cases where you do not want anyone to manipulate the store from the connector level. For example, you want to avoid the chance that someone changes the configuration of a Clear operation so that it deletes all your authorization data.
+Use a private Object Store for:
```

Comment:
Suggestion:
Use a private object store when:

* The shared state...
* You don't want...